src/main/cli/cli.c

```diff
@@ -5064,6 +5064,10 @@ const cliResourceValue_t resourceTable[] = {
 #ifdef USE_INVERTER
     DEFA( OWNER_INVERTER,      PG_SERIAL_PIN_CONFIG, serialPinConfig_t, ioTagInverter[0], SERIAL_PORT_MAX_INDEX ),
 #endif
+#if defined(USE_SOFTSERIAL1) || defined(USE_SOFTSERIAL2)
```

Comment:
Perhaps have a `USE_SOFTSERIAL` definition is either are set which will simplify such conditional statements thereafter.

Reply:
Agree - they are defined both anyway. Updated to `USE_SOFTSERIAL`.